lib/src/review_service/review_prompter_interface.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:in_app_review/in_app_review.dart';
+import 'package:logger/logger.dart';
+
+/// Provides ways to prompt user to review the current application.
+abstract interface class IReviewPrompter {
+  /// Prompts the user to rate the current application using the platform's default application store.
+  Future<void> tryPrompt();
+}
+
+/// Implementation of [IReviewPrompter].
+final class ReviewPrompter implements IReviewPrompter {
+  final Logger _logger;
+
+  ReviewPrompter({
+    required Logger logger,
+  })  : _logger = logger;
+
+  @override
+  Future<void> tryPrompt() async {
+    _logger.d('Trying to prompt user to review the current application.');
+
+    final InAppReview inAppReview = InAppReview.instance;
+
+    if (await inAppReview.isAvailable()) {
+        inAppReview.requestReview();
+    }
+
+    _logger.i('Prompted user to review the current application.');
```

Comment:
Shouldn't this be inside the `if`?
Also, shouldn't we have a different log message when `isAvailable()` returns false?